src/python/pants/backend/codegen/soap/java/extra_fields.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.backend.codegen.soap.target_types import WsdlSourcesGeneratorTarget, WsdlSourceTarget
+from pants.engine.target import StringField
+
+
+class JavaPackageField(StringField):
+    alias = "package"
+    help = "Override destination package for generated sources"
+
+
+class JavaModuleField(StringField):
+    alias = "module"
+    help = "Java module name"
```

Comment:
If we anticipate other languages being generated from WSDL files, the aliases for these fields should probably be prefixed with `java_` or `jvm_`.